crates/storage/src/lazy/mapping.rs

```diff
@@ -161,13 +161,24 @@ where
     ///
     /// Returns `None` if no `value` exists at the given `key`.
     #[inline]
-    pub fn contains<Q>(&self, key: Q) -> Option<u32>
+    pub fn size<Q>(&self, key: Q) -> Option<u32>
```

Comment:
I think `len()` would be more idiomatic?

Reply:
Hmm hmm, not sure because a function `len` on a datastructure `Mapping` invokes different associations and `scale-codec` also uses the `size` terminology.

Reply:
Okay. Just double checking.